README.md

```diff
@@ -44,6 +44,7 @@ cp -r src/ros2_controllers/diff_drive_controller src
 cp -r src/ros2_controllers/imu_sensor_broadcaster src
 rm -rf src/ros2_controllers
 
+pip3 install moms_apriltag # used for apriltag generation
```

Comment:
Why this dependency is not resolved automatically as a dependency of the `panther_docking` package?

Reply:
I cannot find `moms_apriltag` in the rosindex.

Reply:
We need to automatize this anyway. Can we try to incorporate the installation of this dependency into Cmakefile?

Reply:
I run pip command inside cmake